README.md

```diff
@@ -179,7 +184,21 @@ docker-compose up --build
 To log in to the system once it has been locally deployed, enter a username in the designated field and click the
  "Login" button as seen in the figure below.
 
-#### Users & user types
+#### Authentication
+
+The method of authentication for the Rosalution system may vary depending on the deployment environment. In a local
+ development environment, the system utilizes OAuth2 for issuing login tokens. This allows for a streamlined development
+ experience and faster iteration.
+
+In contrast, in a production environment, the system utilizes CAS (Central Authentication Service) for authentication.
+ This means that in order to log in to the production environment, a valid username must be provided, and an OAuth2
+ token will be issued if the username is valid in the system.
+
+It is important to note that the choice of authentication method may be dictated by security and organizational
```

Comment:
This last paragraph can be removed, the first two are sufficient for explaining the production/local auth login.

Reply:
Agree with this. The last paragraph is just summarizing the 1st & 2nd paragraphs, it's not providing more information.

Reply:
I believe I have made this requested change and removed this paragraph